jsonrpsee/src/lib.rs

```diff
@@ -76,6 +76,10 @@ pub use jsonrpsee_types as types;
 #[cfg(any(feature = "http-server", feature = "ws-server"))]
 pub use jsonrpsee_utils::server::rpc_module::{RpcModule, SubscriptionSink};
 
+/// TODO: (dp) any reason not to export this? narrow the scope to `jsonrpsee_utils::server`?
+#[cfg(any(feature = "http-server", feature = "ws-server"))]
```

Comment:
you can remove these feature guards, the `utils` by itself have conditional compilation that should be sufficient but I think you have to change `utils` to be a non-optional dependency for that to work.